postcss.config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "postcss-custom-properties": {
+        "preserve": "true",
+        "warnings": "false"
+    }
+}
```

Comment:
Please include a newline at the end of the file.

I'm not seeing it in the coding guidelines, but it's set in the [`.editorconfig`](https://github.com/Automattic/wp-calypso/blob/110106d322fd8138e9f9d203ac46a0518df2fd82/.editorconfig#L15) & is generally a good practice.

i.e. when you do something like `cat postcss.config.json`, your prompt starts w/ a clean line, etc.

Reply:
Spelled this out in #18927 :)

Reply:
Newline should be there now. Pushed the change ~40min ago but GH is a bit slow today. 
The newline is a default setting in our `prettier` config btw. So all new files should have it by default. So used to it being added on save. Just had temporarily deactivated the option in my IDE that runs `prettier` on save. 🙈